app/forms/steps/check/remove_check_form.rb

```diff
@@ -0,0 +1,22 @@
+module Steps
+  module Check
+    class RemoveCheckForm < BaseForm
+      include SingleQuestionForm
+
+      yes_no_attribute :remove_check
```

Comment:
Validation error (when not selecting any of the radios) is not set in the locales.